packages/editable-commands/package.json

```
@@ -0,0 +1,68 @@
+{
+	"name": "@sapphire/plugin-editable-commands",
+	"version": "0.0.1",
+	"description": "Plugin for @sapphire/framework to have editable commands",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		".": {
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		},
+		"./register": {
+			"import": "./register.mjs",
+			"require": "./register.js"
+		}
+	},
+	"sideEffects": [
+		"./dist/register.js",
+		"./register.js",
+		"./register.mjs"
+	],
+	"homepage": "https://github.com/sapphiredev/plugins/tree/main/packages/editable-commands",
+	"scripts": {
+		"lint": "eslint src tests --ext ts --fix",
+		"build": "tsc -b src",
+		"postbuild": "run-p esm:**",
+		"esm:register": "gen-esm-wrapper dist/register.js dist/register.mjs",
+		"esm:default": "gen-esm-wrapper dist/index.js dist/index.mjs",
+		"prepublishOnly": "yarn build"
+	},
+	"dependencies": {
+		"@skyra/editable-commands": "^2.1.0",
+		"tslib": "^2.3.0"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/sapphiredev/plugins.git",
+		"directory": "packages/editable-commands"
+	},
+	"files": [
+		"dist",
+		"!dist/*.tsbuildinfo",
+		"register.*"
+	],
+	"engines": {
+		"node": ">=14",
+		"npm": ">=6"
```

```suggestion
		"node": ">=16.6",
		"npm": ">=7"
```

to match @skyra/editable-commands / djs